app/controllers/invitations_controller.rb

```diff
@@ -0,0 +1,55 @@
+class InvitationsController < ApplicationController
+  before_action :get_resources, except: [:confirm]
+  skip_before_action :authenticate_user, only: [:confirm]
+
+
+  def new
+    authorize Invitation
+    @invitation = @todo_list.invitations.new
+  end
+
+  def create
+    authorize Invitation
+    @invitation = @todo_list.invitations.new(permitted_attributes(Invitation.new))
+    if @invitation.invited_user_email == current_user.email
```

Comment:
Takie sprawdzenie powinno być częścią walidacji modelu (lub service w późniejszym etapie). Wrzucanie tego do kontrolera powoduje że zwiększa Ci się ilość rzeczy do testowania w tej metodzie oraz w potencjalnych późniejszych. Najlepiej zrobić to tuż obok sprawdzania czy email jest unikalny (bo to ta sama kategoria - odrzucanie rekordu bazując po tym jaki email został wpisany). Zakładam że później zostaje to przeniesione do service (widziałem PR-y) więc spokojnie możemy to tutaj zostawić - piszę raczej żeby przypomnieć.